packages/sku/lib/storybook.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+
+const { paths } = require('../context');
+const fs = require('fs/promises');
+const debug = require('debug');
+const glob = require('fast-glob');
+
+const log = debug('sku:storybook');
+
+const previewFileName = 'preview.{js,ts,tsx}';
+
+const previewFileAbsolutePath = path.join(
+  path.dirname(paths.appSkuConfigPath),
+  '.storybook',
+  previewFileName,
+);
+
+/**
+ * This function first cleans up any existing preview.{js,ts,tsx} files in the provided storybook config directory.
+ * This includes any potentially dangling symlinks.
+ * Then it looks for a `.storybook/preview.{js,ts,tsx}` file relative to the sku config folder.
+ * If 1 file is found, a symlink is created to it from the provided storybook config directory.
```

Comment:
I find **from**s and **to**s confusing in the context of symlinks.
```suggestion
 * If 1 file is found, a symlink pointing to it is created in the provided storybook config directory.
```